packages/wdio-cucumber-framework/src/reporter.js

```diff
@@ -236,6 +236,18 @@ class CucumberReporter {
         return `${name}${line}`
     }
 
+    getTestParent(feature, scenario) {
+        return `${feature.name || 'Undefined Feature'} > ${scenario.name || 'Undefined Scenario'}`
+    }
+
+    getTestStepTitle(step) {
+        return ((step.keyword || '') + (step.text || 'Undefined Step')).trim()
+    }
+
+    getTestFullTitle(parent, stepTitle) {
+        return `${parent} > ${stepTitle}`
+    }
```

Comment:
This should be all small utility methods and don't belong to the class

Reply:
Won't be able to do it today

Reply:
@christian-bromann done